.changeset/strange-drinks-sip.md

```diff
@@ -0,0 +1,5 @@
+---
+'@urql/storage-rn': minor
+---
+
+Bumped RN Netinfo and Async Storage to latest version and added Netinfo v11 to peer dependency array
```

Comment:
```suggestion
Add `@react-native-community/netinfo@^11.0.0` to the list of supported peer dependencies
```

Just a quick note here ✌️ These are user-facing, so from a user’s perspective the only relevant change is the newly supported peer dependency, since `devDependencies` are only for ourselves (i.e. this repo)

I just had a quick look at whether any of the major versions had an actionable change for us, but it looks fine 👍